packages/mds-policy-author/api.ts

```diff
@@ -188,6 +189,10 @@ function api(app: express.Express): express.Express {
 
   app.post(pathsFor('/policies'), async (req, res) => {
     const policy = req.body
+    if (!isUUID(policy.policy_id)) {
+      /* eslint-disable-next-line no-param-reassign */
+      policy.policy_id = uuid.v4()
```

Comment:
This is the _only_ place in the code that uses `.v4()`; everywhere else just uses `uuid()`